From 2a98868c1defceb2abdf529c225624ce9dd23fb9 Mon Sep 17 00:00:00 2001
From: Michal Hrusecky <michal.hrusecky@nic.cz>
Date: Wed, 5 Dec 2018 10:45:03 +0100
Subject: [PATCH] Revert "rescue-image: Fix wget dependency"

This reverts commit 33e88b04f00c1d0ae1e68b481c3b9b639510a959.

This applies only to OpenWRT master
---
 cznic/rescue-image/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cznic/rescue-image/Makefile b/cznic/rescue-image/Makefile
index c82fab827..2104c22cb 100644
--- a/cznic/rescue-image/Makefile
+++ b/cznic/rescue-image/Makefile
@@ -17,7 +17,7 @@ PKG_SOURCE_URL:=https://www.busybox.net/downloads \
 PKG_HASH:=0a367e19cdfd157e8258d87f893ee516
 PKG_MAINTAINER:=Michal Hrusecky <michal.hrusecky@nic.cz>
 PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)
-ROOTFS_INSTALL:=+libgcc +libc +libubox +mtd +libpthread +librt +libattr +libuuid +zlib +libblkid +liblzo +btrfs-progs +schnapps +dropbear +uboot-envtools +usign +cznic-repo-keys +libopenssl +libpcre2 +wget
+ROOTFS_INSTALL:=+libgcc +libc +libubox +mtd +libpthread +librt +libattr +libuuid +zlib +libblkid +liblzo +btrfs-progs +schnapps +dropbear +uboot-envtools +usign +cznic-repo-keys +libopenssl +libpcre +wget
 PKG_BUILD_DEPENDS:=opkg/host mtd btrfs-progs schnapps wget cznic-repo-keys usign ubooot-envtools dropbear
 
 KERNEL_BUILD_DIR:=$(PKG_BUILD_DIR)/kernel
-- 
2.19.2

